pkg/services/ngalert/state/manager.go

```diff
@@ -187,7 +188,7 @@ func (st *Manager) DeleteStateByRuleUID(ctx context.Context, ruleKey ngModels.Al
 		if s.State != eval.Normal {
 			startsAt = now
 		}
-		s.SetNormal(reason, startsAt, now)
+		s.SetNormal(reason, NoCause, startsAt, now)
```

Comment:
I would change this to `CauseNone` as then all cause enums have the same prefix.

```suggestion
		s.SetNormal(reason, CauseNone, startsAt, now)
```